lib/PuppeteerSharp/Cdp/CdpPage.cs

```diff
@@ -1309,4 +1310,25 @@ private Task ResetBackgroundColorAndViewportAsync(ScreenshotOptions options)
             : Task.CompletedTask;
         return Task.WhenAll(omitBackgroundTask, setViewPortTask);
     }
+
+    private void AttachExistingTargets()
+    {
+        List<ITarget> queue = [];
+        queue.AddRange(_targetManager.GetChildTargets(PrimaryTarget));
+
+        var idx = 0;
+        while (idx < queue.Count)
+        {
+            var next = queue[idx] as CdpTarget;
+            idx++;
+            var session = next!.Session;
```

Comment:
Should we use a hard cast instead of adding `!`?

Reply:
Yup